ui/eureka/eureka_config.ts

```diff
@@ -38,6 +38,7 @@ UserConfig.registerOptions('eureka', {
       id: 'CompleteNamesSTQ',
       name: {
         en: 'Prefer complete names for Skirmishes/Critical Engagements',
+        de: 'Bevorzuge komplette Namen für Scharmützel/Kritische Gefechte',
```

Comment:
Kritische -> kritische

Reply:
tbh i would leve it capitalised as it is its own name which is also refered to as KG in many groups